tests/spread/integration/apt/test_mini_install_ocaml.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+if [[ "$1" != "--spread" ]]; then
+    FILE_DIR=$(realpath "$(dirname "$0")")
+    source "$FILE_DIR"/setup.sh
+fi
+
+## TESTS 
+# spellchecker: ignore rootfs resolv localtime postinst tzdata zoneinfo
+rootfs="$(install-slices apt_apt-get-mini)"
+
+# For DNS resolution
+cp /etc/resolv.conf "$rootfs/etc/resolv.conf"
+
+# # Ocaml wants /dev/null
```

Comment:
Please put a note about why these are commented out.

Reply:
they were not supposed to be actually. ty for the catch